index.html

```diff
@@ -88,6 +104,9 @@
     ["Dynkin system"],
     ["sigma-algebra"],
     ["ultrafilter"],
+    ["Cayley-graph"],
```

Comment:
I think this should be written without the hyphen? e.g. https://en.wikipedia.org/wiki/Cayley_graph